src/services/user/adminBlockUser.js

```diff
@@ -1,12 +1,18 @@
 const {successResponse, errorResponse} = require('../../utils');
 const {adminBlockUserProcess} = require('../../processes/scoring/admin-block-unblock-user');
+const UsersFunction = require('../../databases/functions/users');
 
 const adminBlockUser = async (req, res) => {
   try {
     const {userId} = req.body;
     if (!userId) {
       return errorResponse(res, 'userId is required', 400);
     }
+    try {
+      await UsersFunction.userBlockByAdmin(userId);
+    } catch (error) {
+      return errorResponse(res, 'Failed to unblock user by admin', 500);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error message in the catch block is misleading. It says 'Failed to unblock user by admin' while it should be 'Failed to block user by admin'.
```diff
- return errorResponse(res, 'Failed to unblock user by admin', 500);
+ return errorResponse(res, 'Failed to block user by admin', 500);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->